src/composables/useIsMobile/index.ts

```diff
@@ -24,24 +26,17 @@ window.addEventListener('resize', () => {
 /**
  * Use global isMobile state
  *
- * @return {import('vue').DeepReadonly<import('vue').Ref<boolean>>} Reactive flag for MOBILE_BREAKPOINT
+ * @return Reactive flag for MOBILE_BREAKPOINT
  */
-export function useIsMobile() {
+export function useIsMobile(): Readonly<Ref<boolean>> {
```

Comment:
Doesn't TS infer type correctly here?

Reply:
it does but explicit types are recommended to fail hard if dependencies change